.github/workflows/ci.yml

```diff
@@ -68,8 +68,10 @@ jobs:
           architecture: 'x64'
       - name: Install pylint
         run: cat dev_tools/conf/pip-list-dev-tools.txt | grep "pylint" | grep -v "#" | xargs pip install
+      - name: Display version
+        run: check/pylint --version
```

Comment:
What's wrong with pinning an exact version of pylint instead?

Reply:
This is a fair point - the `pylint~=2.6.0` in `pip-list-dev-tools.txt` can get any version of the form `2.6.*`.

That being said, I've definitely seen discrepancies between pylint runs with the same pylint version. I don't think pinning it will resolve the flake.